### DISCUSSION TITLE
[bug] "Default Title" option on products with no variants

### DISCUSSION THREAD

**Discussion by matt-meagher** [Shopify/storefront-api-feedback#198] [Help]
Hello!

We use the storefront api for a couple of our products and have noticed this issue across all of them.
Products with no variants assigned to them return an option: 
```
 {
    "name": "Title",
    "values": [
       "Default Title"
    ]
}
```

Adding a variant will remove this, but once the added variant is removed from the product the "Default Title" option is added back. 

Of course we could just look for this value in our front end code and not render the option picket, but it doesn't seem like a great long term solution. 
Looking on Google it looks like this had been an issue on Liquid themes in the past (I could not reproduce on a liquid theme).

Please let me know if this is not a bug and I am maybe missing something in configuration. I can't imagine we are the only ones with products with 0 options using the storefront api.

I appreciate it.

![Screenshot 2023-07-13 at 9 01 59 AM](https://github.com/Shopify/storefront-api-feedback/assets/112413786/3f8b5845-2226-417f-b7e3-61a215cb201d)
![Screenshot 2023-07-13 at 8 53 57 AM](https://github.com/Shopify/storefront-api-feedback/assets/112413786/3967f0a4-a0bc-48dd-bc5c-dcd31dc82981)

**Comment by Felixmannen123** [comment]
I'm having the same issue, also due to importing products with the API. Not found a solution yet.

**Comment by mihaiav** [comment]
I have the same issue. I can't believe Shopify has such a bug...The worst part is that you can't fix it.

**Comment by joaovpmamede** [comment]
I was just testing hydrogen and this happened to me.
What's even worse is that if I render a product (route) that has no variants I get that `Default Title` on a query string added automatically to my url (e.g. `http://example.com/products/product-name?Title=Default+Title`)
